diff --git a/src/schema.rs b/src/schema.rs
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -1,5 +1,5 @@
 table! {
     active_subscriptions (id) {
-        id -> Integer,
+        id -> BigInt,
         expires_at -> BigInt,
     }
@@ -8,6 +8,6 @@ table! {
 table! {
     renewing_subscriptions (new) {
-        old -> Integer,
-        new -> Integer,
+        old -> BigInt,
+        new -> BigInt,
     }
 }
@@ -15,5 +15,5 @@ table! {
 table! {
     subscriptions (id) {
-        id -> Integer,
+        id -> BigInt,
         hub -> Text,
         topic -> Text,
